### ISSUE TITLE
restart browser with each describe

### ISSUE THREAD

**Issue by shaoping** [angular/protractor#1785]
now we have the restart browser with each test , it costs so much times.
```
// If true, protractor will restart the browser between each test. 
  // CAUTION: This will cause your tests to slow down drastically.
  restartBrowserBetweenTests: false,
```

can we restart browser with each describe or with each file

#1435

**Comment by ilanbiala** [comment]
+1 please add this!

**Comment by nishakchaudhari** [comment]
did you find any solution to the above issue? @shaoping

**Comment by shaoping** [comment]
@nishakchaudhari   did not have useful solution for this， i  execute the describe with more config files

**Comment by omarazmi** [comment]
+1

**Comment by eddyoc** [comment]
+1 after each suite makes sense. This would assist our CI with no end

**Comment by borisdiakur** [comment]
:+1: for each suite fresh start

**Comment by adiehl** [comment]
+1

**Comment by chetanb** [comment]
Have we achieved this?

**Comment by Warun26** [comment]
I could make it work with Jasmine 2 framework. I emit an event when a describe finishes. Will make a pull request in the weekend

**Comment by shaoping** [comment]
please achieve this , as the test increase biger, it's a big problem

**Comment by MisterSpicy** [comment]
+1, this would be quite nice.

**Comment by bhalajin** [comment]
+1
This would make sense because the related tests would be completed at a go before restarting the browser.

**Comment by ilanbiala** [comment]
@angular why is this in the icebox if this is a really important feature for not having slow tests?

**Comment by NickTomlin** [comment]
@ilanbiala Please feel free to submit a PR if it is something that you really need.

**Comment by ilanbiala** [comment]
@NickTomlin do you disagree?

**Comment by NickTomlin** [comment]
@ilanbiala I personally don't use milestones as a way of tracking features, so I don't have strong opinions on this being in the icebox or not. I do feel that there's no sense in moving it out unless someone wants to champion the feature.

**Comment by avdd** [comment]
So much noise, when the solution is so simple.  The original fix for #1435 should have made this general from the start.
